HW0/P4/P4.py

```diff
@@ -30,47 +30,92 @@
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
     #####################
+    s_true=np.loadtxt("P4_trajectory.txt", delimiter=",")
+    x_coords=s_true[:,0]
+    y_coords=s_true[:,1]
+    z_coords=s_true[:,2]
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '--b', label='True trajectory')
+    ax.plot(x_coords, y_coords, z_coords,
+            '--b', label='True trajectory')
 
     #####################
     # Part 2:
     #
     # Read the observation array and plot it (Part 2)
     #####################
-
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    m = np.loadtxt("P4_measurements.txt", delimiter=",")
+    C = np.array([[1./rx, 0.0, 0.0],
+                  [0.0, 1./ry, 0.0],
+                  [0.0, 0.0, 1./rz]])
+    s_obs=np.dot(C, m.T)
```

Comment:
Nice! matrix operations are faster than vector operations in a for loop

I would be careful with naming this matrix C though because C in the context of this problem is actually another matrix